explorer-api/src/mix_node/models.rs

```diff
@@ -10,7 +10,7 @@ use std::sync::Arc;
 use std::time::SystemTime;
 use tokio::sync::RwLock;
 
-#[derive(Clone, Debug, Serialize, JsonSchema)]
+#[derive(Clone, Debug, Serialize, JsonSchema, PartialEq)]
 #[serde(rename_all = "snake_case")]
 pub(crate) enum MixnodeStatus {
```

Comment:
This is not a comment regarding this PR, but more of a mental note that currently we have enums in 3 different places doing essentially the same thing and we should unify them at some point...

Reply:
Yeah, I tried to re-use the other ones, they were just lacking `JsonSchema` :cry: